source/vir/src/early_exit_cf.rs

```diff
@@ -0,0 +1,161 @@
+use crate::ast::{ArmX, CallTarget, Expr, ExprX, Stmt, StmtX, VirErr};
+use air::ast::Span;
+use air::errors::error;
+
+#[derive(Copy, Clone, Debug)]
+enum StatementType {
+    Return,
+    //Break,
```

Comment:
Are these commented out because they're currently unsupported? Can you add a TODO explaining that?